guides/v2.1/cloud/release-notes/cloud-tools.md

```diff
@@ -18,14 +18,84 @@ You can list the available ece-tools commands using:
+
+   -  JIRA--MAGECLOUD-2156-->Improved validation of environment variables to identify configuration settings that can cause conflicts during the {{site.data.var.ece}} build, deploy, and post-deploy phases. For example, a warning message displays during the install and upgrade process if the global setting for static site content deployment conflicts with settings on the build or deploy phase.
+
+-  **Environment variable updates**—Changed the following environment variables:
+
+    -  JIRA--MAGECLOUD-2435-->**[SKIP_HTML_MINIFICATION global variable]({{ page.baseurl }}/cloud/env/variables-intro.html##skip_html_minification)**—Changed the default value from `false` to `true` to enable the on-demand static content minification setting in the default environment configuration. This change decreases downtime significantly and is required for zero-downtime deployments. See [Global variables]({{ page.baseurl }}/cloud/env/variables-intro.html##skip_html_minification).
+
+    -  JIRA--MAGECLOUD-1506-->**[CLEAN_STATIC_FILES deploy variable]({{ page.baseurl }}/cloud/env/variables-deploy.html#clean_static_files)**—Added the capability to clean static content files generated during the build phase and updated the default to clean static content by default (``CLEAN_STATIC_FILES=`true``). Previously, you could only clean static content files generated during the deploy phase.
+
+-  **Logging**-Made the following changes to improve log messages and reduce log size:
+
+   -  JIRA--MAGECLOUD-2489-->Deployment failure log entries now include the command output from the operations that cause the failures.
+
+   -  JIRA--MAGECLOUD-2209-->Added logging for deployment failures that occur when the generated factories required by some extensions cannot be generated correctly because the file system is in a read-only state.
+
+   -  JIRA--MAGECLOUD-2402-->Improved the deploy log to reduce log size and fix formatting issues caused by setup commands that used the interactive progress bar.
+
+   -  JIRA--MAGECLOUD-2227-->Eliminated unnecessary verbosity and updated the priority levels for some log statements.
+
+
+#### Resolved Issues
+
+-  **Cron-specific fixes**
+
+   -  JIRA--MAGECLOUD-2427-->Changed the default settings for cron job history lifetime from 3d to 1h to prevent performance issues and deployment failures that can occur when the cron queue fills too quickly.
+
+   -  JIRA--MAGECLOUD-2508-->Fixed an issue with the [`cron:unlock`]({{ site.baseurl }}/guides/v2.2/cloud/trouble/reset-cron-jobs.html) command so that it works in {{site.data.var.ee}} v2.1. Previously, the command was supported only in v2.2.
```

Comment:
This is not only fixing the CLI command but is also running equivalent process during deployment on deploy phase to reset cron schedule & jobs after deployment is finished.

Reply:
I think the cron scheduling changes on the deploy phase are covered in the following release item in  the New features section: 

```-  JIRA--MAGECLOUD-2445--> **Cron scheduling improvements**—Improved the cron job management process during the deploy phase to prevent database locks and other critical issues. Now, all cron jobs stop during the deploy phase and restart after the deploy phase completes.``

[MAGECLOUD-2445](https://magento2.atlassian.net/browse/MAGECLOUD-2445)